content/api/alerts.apib

```diff
@@ -0,0 +1,205 @@
+FORMAT: 1A
+title: Alerts
+description: Configure alerts for your SparkPost account.
+label: Labs
+
+# Group Alerts
+Alerts can be used to get notifications regarding various activities on your SparkPost account.
+
+Alerts can be configured to send notifications via email to up to 10 email addresses, and supports the following activity:
+* Approaching monthly sending limit
+
+Alerts are configured to be account-wide. Currently, subaccount level alerts are not supported. 
+
+## Alert Object
+
++ Data Structure: Attributes
+    + name (string) - Display name of the alert.
+    + description (string) - Description of the alert.
+    + id (string) - Unique alphanumeric id.
+    + email_addresses (array) - List of email addresses to send alert notifications.
+    + alert_metric (enum) - Metric that the alert is configured to track.
+        + `"monthly_sending_limit"`
+    + threshold (object) - Configuration for level and threshold at which an alert is generated.
+        + error (object) - Error level alert configuration.
+            + target (number) - Percentage value at which an alert will be generated.
+            + comparator (enum) - Conditional comparison operator used by alerts service to compare actual value of `alert_metric` to the `target` value.
+                + `"gt"`
+    + enabled (boolean) - Whether or not the alert is active.
+
++ Sample
+        
+    ```
+    {
+        "name": "Sparky's Monthly Alert",
+        "description": "Example alert that will send a notification when 90% of monthly sending limit is used",
+        "id": "cbe0faf0-9051-11e8-ad04-490e82764b23",
+        "email_addresses": [
+            "sparky@sparkpost.com"
+        ],
+        "alert_metric": "monthly_sending_limit",
+        "threshold": {
+            "error": {
+                "target": 90,
+                "comparator": "gt"
+            }
+        },
+        "enabled": true
+    }
+    ```
+
+### Create an Alert [POST /labs/alerts]
+Creates a new alert
+
++ Data Structure
```

Comment:
What is required here?

Reply:
whoopsie, updating. it's `name`, `email_addresses`, `alert_metric`, `threshold.error.target` :)